simularium/simularium-website
# Feature/conversion title

Time estimate or Size
=======
_small_
Will get review at UX office hours prior to merging, code ready for review.

Problem
=======
Advances #396 
Closes: #509 

Solution
========
UI for input, and optional property on `ConversionProcessingData` for title.

Small tweaks to the naming of `fileName` and `backendFileName` for clarity in the file conversion logic.

Steps to Verify:
----------------
1. Send a Smoldyn file for conversion and provide a title, check that the title appears in the trajectory you receive back.
(Tip: logging the data on line 271 in `ViewerPanel.tsx` in `handleIncomingConvertedFile`)

Screenshots (optional):
-------------------
![Screenshot 2024-09-16 at 9 20 06 AM](https://github.com/user-attachments/assets/b7cce272-4bf2-42e2-8e4a-afc7578ab880)
